integreat_cms/cms/views/push_notifications/push_notification_form_view.py

```diff
@@ -263,6 +263,14 @@ def post(self, request, *args, **kwargs):
             for form in pnt_formset:
                 if not form.is_valid():
                     form.add_error_messages(request)
+        elif set(pn_form.cleaned_data["regions"]).difference(request.available_regions):
+            logger.warning(
+                "%r is not allowed to post news to %r (allowed regions: %r)",
+                request.user,
+                push_notification_instance,
+                request.available_regions
+            )
```

Comment:
This message only makes sense with `pn_form.cleaned_data["regions"]`:
```suggestion
            logger.warning(
                "%r is not allowed to post news to %r (allowed regions: %r)",
                request.user,
                pn_form.cleaned_data["regions"],
                request.available_regions,
            )
```

Reply:
True, I mixed up the parameters.